src/app/sections/footer/footer.component.html

```diff
@@ -0,0 +1,24 @@
+
+<footer>
+  <section class="containerFooter">
```

Comment:
Footer is a wrapper, u don't need a section.

```suggestion
<footer class="containerFooter">
```